kod/object/active/holder/nomoveon/battler/monster/towns/hazartwn/hzapoth.kod

```diff
@@ -59,7 +59,7 @@ messages:
       return False;
    }
 
-   UserEntered(who = $)
+   UserEnter(who = $)
```

Comment:
I am confused as to why some UserEntered became WelcomeUser, and some became UserEnter.